iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/VscConverterStationAdapter.java

```diff
@@ -17,57 +17,61 @@ protected VscConverterStationAdapter(final VscConverterStation delegate, final M
         super(delegate, index);
     }
 
-    // -------------------------------
-    // Not implemented methods -------
-    // -------------------------------
     @Override
-    public ReactiveLimits getReactiveLimits() {
-        throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+    public TerminalAdapter getTerminal() {
```

Comment:
Are you sure about the return type (`Terminal` vs `TerminalAdapter`)

Reply:
I will change all return Adapter<I> type to I